packages/terra-infinite-list/src/terra-dev-site/test/infinite-list/InfiniteListVirtual.test.scss

```diff
@@ -0,0 +1,11 @@
+@import './InfiniteListTestCommon';
+
+:local {
+  .item-content {
+    @include content-common(100px);
```

Comment:
lets drop the @include syntax and add the styles directly

Reply:
Done